troubleshooting.md

```diff
@@ -143,3 +143,21 @@ If you are using a useradd command within a Dockerfile with a large UID/GID, it
 #### Solution
 
 If the entry in the Dockerfile looked like: RUN useradd -u 99999000 -g users newuser then add the `--log-no-init` parameter to change it to: `RUN useradd --log-no-init -u 99999000 -g users newuser`. This option tells useradd to stop creating the lastlog file.
+
+### 7) Permission denied when running Podman commands
+
+When rootless podman attempts to execute a container on a non exec home directory a permission error will be raised.
+
+#### Symptom
+
+If you are running podman or buildah on a home directory that is mounted noexec,
+then they will fail. With a message like:
+
+```
+podman run centos:7
+standard_init_linux.go:203: exec user process caused "permission denied"
+```
+
+#### Solution
+
+Since the administrator of the system setup your home directory to be noexec, you will not be allowed to execute containers from storage in your home directory. It is possible to work around this by manually specifying a container storage path that is not on a noexec mount. Simply copy the file /etc/containers/storage.conf to ~/.config/containers/ (creating the directory if necessary). Specify a directory which is not on a noexec mount and you user have read/write privileges to for the graphroot.  You will need to modify other fields to writable directories as well.
```

Comment:
Should tell them to set `graphroot` and `runroot` specifically. And probably to set `driver` to VFS so they don't try using normal overlayfs as non-root